app/views/donuts/show.html.erb

```diff
@@ -9,7 +9,11 @@
   <div class="donut-card d-flex shadow-sm flex-row bg-white">
     <div class="p-4 col-6 d-flex flex-column align-items-center">
       <div class="w-100 py-4 text-center">
+        <% if @donut.photo.key %>
+          <%= cl_image_tag donut.photo.key, crop: :fill, class: "donut-image w-75 shadow-sm" %>
```

Comment:
should this be @donut rather than donut?

Reply:
also the .donut-image class will make it a fixed 250x250 round image, not sure if you want that